examples/activate-mongodb-atlas-resources.template.yaml

```
@@ -0,0 +1,339 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MongoDB Resource activation - This template can be used to activate all MongoDB Atlas resources in current AWS account in the selected region.
+Metadata:
+  cfn-lint: { config: { ignore_checks: [ W9002, W9003, W9006, E3001, E1010 ] } }
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: AWS Region
+        Parameters:
+          - Region
+Parameters:
+  Region:
+    Description: The AWS Region where resources would be activated.
```

If this is a description for the user modifying the template (please correct me if I've misunderstood) I think this could be clearer that these are regions for them to configure as needed:

```suggestion
    Description: The AWS Regions where resources will be activated.
```

---

The user can select only one region at a time in this template

---

updated to `The AWS Region where resources will be activated.`